cmake/BLTOptions.cmake

```diff
@@ -143,6 +143,11 @@ set(BLT_CODE_STYLE_TARGET_NAME "style" CACHE STRING "Name of the master code for
 set(BLT_DOCS_TARGET_NAME "docs" CACHE STRING "Name of the master documentation generation target")
 set(BLT_RUN_BENCHMARKS_TARGET_NAME "run_benchmarks" CACHE STRING "Name of the target to run benchmark tests")
 
+# The following variables control whether third-party libraries (TPLs) should have targets exported from  
+# a project using BLT, or if the the TPLs should be configured downstream and have their setup CMake files 
+# installed.  These are set by calling either blt_install_tpl_setups or blt_export_tpl_targets.
+set(EXPORT_TPL_TARGETS FALSE)
+set(INSTALL_TPL_SETUPS FALSE)
```

Comment:
This file is for user facing options and these two variables are for internal sanity check. Please move them into BLTMacros.cmake by the macros they are used in. 

Also prefix them with `BLT_` so they don't clash with project variables.  Also mark them advanced so they don't show up in `ccmake`.

Reply:
Will do! Thanks Chris